Documentation/cheatsheet.rst

```diff
@@ -152,26 +152,12 @@ Policy management
 -----------------
 
 
-Importing a Cilium Network Policy
-
-.. code-block:: shell-session
-
-    cilium-dbg policy import my-policy.json
-
-
 Get list of all imported policy rules
 
 .. code-block:: shell-session
 
     cilium-dbg policy get
 
-Remove all policies
-
-.. code-block:: shell-session
-
-    cilium-dbg policy delete --all
-
-
```

Comment:
Is the subsection title still needed? I think listing the imported policy rules could be merged to the previous subsection (“Basics”).